ui/js/directives/treeherder/clonejobs.js

```diff
@@ -239,9 +239,22 @@ treeherder.directive('thCloneJobs', [
 
                 addJobBtnToArray(job, lastJobSelected, jobBtnArray);
             }
+            return jobBtnArray;
+        };
+        var addJobBtnEls = function(jgObj, jobList) {
```

Comment:
I think it might be better to just remove this function and inline the code directly in the two places that it's called, since it's so simple.